### ISSUE TITLE
Make the output quality configurable for JPEG images

### ISSUE THREAD

**Issue by Linden6** [lovasoa/dezoomify-rs#66]
Firstly, many thanks for this wonderful tool.  

I would be grateful for any help with enhancing the tool to take these two issues into account:

## Jpeg Quality
I've noticed that dezoomify-rs saves jpeg's with a quality of 75%.  As I prefer to save images with less loss of quality I have hacked my local copy of encoder/canvas.rs to set a jpeg quality of 90% by deleting:
`    self.image.save(&self.destination).map_err(|e| {`
and, replacing it with:
```rust
    let fout = &mut BufWriter::new(File::create(&self.destination)?);	
    let mut j = image::jpeg::JPEGEncoder::new_with_quality(fout, 90);
    let ib = &self.image;
    j.encode(&ib, ib.width(), ib.height(), image::ColorType::Rgba8).map_err(|e| {
```
[Which, requires the following use declarations above:
```rust
    use std::io::{BufWriter};
    use std::fs::File;
```
]
    
However, I am not familiar with Rust and there may be a better way of achieving this.  Ideally the quality would be set via a flag when calling the tool from the command line.

**Comment by lovasoa** [comment]
Let's track the JPEG quality setting feature here, and create a new one for color profiles.

**Comment by lovasoa** [comment]
I added a `--compression` command-line argument. You can now call dezoomify with `--compression 10` to set the image quality to 90 (the quality is computed as 100-compression). The new compression parameter is also used in the PNG encoder, where it can be set to a lower value on less performant hardware to speed up the image creation at the expense of a small file size increase.

@Linden6 : Can you help test the new feature ? If everything works as expected, I'll release a new version. The latest development builds of the software can be downloaded [from github actions](https://github.com/lovasoa/dezoomify-rs/actions?query=workflow%3A%22Master+optimized+builds%22).

**Comment by Linden6** [comment]
Many thanks for your speedy response.

I've just tested the latest build (Build 133224ea6f01a2213763eec3ac6bb6504a4a95f7) via the pre-compiled Linux version.  I've downloaded the same image a couple of times and saved it as a JPEG with different compression settings.  At a compression value of 90 the image quality was visually very poor compared to a compression value of 10, and the file was also about one tenth the size - which is what one would expect.  I've also used the ImageMagick 'identify' command line tool to confirm the quality, with values returned as expected.

Thanks again!
